pkg/printers/humanreadable.go

```diff
@@ -785,12 +772,12 @@ func appendLabelCells(values []interface{}, itemLabels map[string]string, opts P
 
 // FormatResourceName receives a resource kind, name, and boolean specifying
 // whether or not to update the current name to "kind/name"
-func FormatResourceName(kind, name string, withKind bool) string {
-	if !withKind || kind == "" {
+func FormatResourceName(kind schema.GroupKind, name string, withKind bool) string {
+	if !withKind || kind.Empty() {
 		return name
 	}
 
-	return kind + "/" + name
+	return strings.ToLower(kind.String()) + "/" + name
```

Comment:
The string will include non-empty `Group` name as well, right? Do we really want it like this?

Reply:
Yes. That matches the format already used by `-o name`. If the group is non-empty, the kind by itself is ambiguous